working-with-fhir-igs/hl7v2-to-dk-medcom/data_mappings.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/uuid;
```

Comment:
Add licence, check other places

Reply:
fixed in [4a378ec](https://github.com/ballerina-guides/healthcare-samples/pull/38/commits/4a378ec4f64c338c69ab412bcd053c7d95901093)